packages/bot-skeleton/src/services/api/active-symbols.js

```diff
@@ -6,7 +6,7 @@ export default class ActiveSymbols {
     constructor(ws, trading_times) {
         this.active_symbols = [];
         this.disabled_symbols = ['frxGBPNOK', 'frxUSDNOK', 'frxUSDNEK', 'frxUSDSEK']; // These are only forward-starting.
-        this.disabled_submarkets = ['energy', 'step_index'];
+        this.disabled_submarkets = ['energy'];
```

Comment:
```suggestion
        this.disabled_submarkets = config.DISABLED_SUBMARKETS;
```

Lets check it together where we can utilize the common config @farabi-deriv

Reply:
@sandeep-deriv yes sure. ✅